vocabularies/Common.xml

```diff
@@ -903,8 +903,8 @@ The minimum structure is
       <!-- Section: Fiscal Points in Time -->
       <Term Name="IsFiscalYear" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property">
         <Annotation Term="Core.Description">
-          <String>Property encodes a fiscal year number as string following the logical pattern YYYY(Y*) consisting of at least four digits. 
-          The string matches the regex pattern [1-9][0-9]{3,}
+          <String>Property encodes a fiscal year number as string following the logical pattern YYYY consisting of four digits. 
```

Comment:
I think it's fine to restrict ourselves to four digits at the moment. If the definition makes clear that this is a year, the upcoming extension to more than four digits is clear to anyone, so I would not consider this an incompatible change.